src/frontend/src/components/ProjectDetailsV2/ActivitiesPanel.tsx

```diff
@@ -18,11 +19,11 @@ const sortByList = [
 
 const ActivitiesPanel = ({ defaultTheme, state, params, map, view, mapDivPostion, states }) => {
   const displayLimit = 10;
-  const [searchText, setSearchText] = useState('');
-  const [taskHistories, setTaskHistories] = useState([]);
+  const [searchText, setSearchText] = useState<string>('');
```

Comment:
you don't have to specify string for usestate if you set the initial state to be a string with quotation